base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -211,12 +213,12 @@ impl ConsensusConstants {
     }
 
     /// Maximum transaction weight used for the construction of new blocks. It leaves place for 1 kernel and 1 output
+    /// with default features
     pub fn get_max_block_weight_excluding_coinbase(&self) -> u64 {
-        self.max_block_transaction_weight - self.coinbase_weight()
+        self.max_block_transaction_weight - self.calculate_1_output_kernel_weight()
     }
 
-    pub fn coinbase_weight(&self) -> u64 {
-        // TODO: We do not know what script, features etc a coinbase has - this should be max coinbase size?
+    fn calculate_1_output_kernel_weight(&self) -> u64 {
```

Comment:
should we not just pass in the OutputFeatures and script?

Reply:
The wallet calculate thats details only after it gets a template from the miner. 
To get the template we need to know how large it will be beforehand. 
Its a potential bug atm I logged it in the audit findings. 
But for this PR I only changed to name of the function to correctly state what it does